diff -r ffdac52d215a Makefile
--- a/Makefile	Fri Apr 12 15:17:15 2013 +1000
+++ b/Makefile	Tue Apr 23 13:48:15 2013 +0200
@@ -58,7 +58,7 @@
 build_extras:
 	$(INSTALL) MySQL-python
 	$(INSTALL) recaptcha-client
-	$(INSTALL) wsgiproxy
+	$(INSTALL) wsgiproxy2
 	$(INSTALL) wsgi_intercept
 	$(INSTALL) "python-ldap == 2.3.13"
 	$(INSTALL) coverage
diff -r ffdac52d215a dev-reqs.txt
--- a/dev-reqs.txt	Fri Apr 12 15:17:15 2013 +1000
+++ b/dev-reqs.txt	Tue Apr 23 13:48:15 2013 +0200
@@ -13,7 +13,7 @@
 SQLAlchemy <= 0.7.999
 PyMySQL
 pymysql_sa
-WSGIProxy
+WSGIProxy2
 recaptcha-client
 metlog-py >= 0.9.5
 metlog-cef
diff -r ffdac52d215a prod-reqs.txt
--- a/prod-reqs.txt	Fri Apr 12 15:17:15 2013 +1000
+++ b/prod-reqs.txt	Tue Apr 23 13:48:15 2013 +0200
@@ -13,7 +13,7 @@
 SQLAlchemy == 0.7.9
 PyMySQL == 0.5
 pymysql_sa == 1.0
-WSGIProxy == 0.2.2
+WSGIProxy2 == 0.1
 recaptcha-client == 1.0.6
 metlog-py == 0.9.8
 metlog-cef == 0.2
diff -r ffdac52d215a services/tests/support.py
--- a/services/tests/support.py	Fri Apr 12 15:17:15 2013 +1000
+++ b/services/tests/support.py	Tue Apr 23 13:48:15 2013 +0200
@@ -223,7 +223,7 @@
     If TEST_REMOTE is set in the environ, will run against a real server.
     """
     import urlparse
-    from wsgiproxy.exactproxy import proxy_exact_request
+    from wsgiproxy2.exactproxy import proxy_exact_request
     from webtest import TestApp
 
     # runs over a proxy
diff -r ffdac52d215a setup.py
--- a/setup.py	Fri Apr 12 15:17:15 2013 +1000
+++ b/setup.py	Tue Apr 23 13:48:15 2013 +0200
@@ -38,7 +38,7 @@
 from setuptools import setup, find_packages
 
 install_requires = ['SQLAlchemy', 'Paste', 'PasteDeploy', 'WebOb',
-                    'Routes', 'simplejson', 'cef', 'wsgiproxy', 'metlog-py']
+                    'Routes', 'simplejson', 'cef', 'wsgiproxy2', 'metlog-py']
 
 
 # extracting the version number from the .spec file
